js/arena.js

```diff
@@ -0,0 +1,50 @@
+export default class Arena {
+    constructor(name){
+      this.name = name.toUpperCase() 
+       this.gladiators = []
+    }
+  
+    addGladiator(gladiator){
+      if(this.gladiators.length < 2){
+        this.gladiators.push(gladiator)
+      } 
+      else {
+        console.log("never have more than 2 gladiators in it at a time")
+       }
+    }
+
+    fight(){
```

Comment:
fight function doesn't work. it doesn't save the result in the gladiators array.